src/Student.js

```diff
@@ -0,0 +1,11 @@
+class Student {
+  constructor(studentId = new Date(), firstName, lastName, githubName, email) {
```

Comment:
Dates don't make for very good ID's. What if two students are created at exactly the same time?